src/main/java/io/github/railroad/utility/ConfigHandler.java

```diff
@@ -0,0 +1,100 @@
+package io.github.railroad.utility;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.google.gson.JsonObject;
+import org.json.*;
+public class ConfigHandler {
+
+    public ConfigHandler() {}
+    public void CreateDefaultConfigs() {
+        CreateDirectory(getConfigPath());
+        createProjectsJsonIfNotExists();
+    }
+
+
+    private void createProjectsJsonIfNotExists() {
```

Comment:
```suggestion
import org.json.*;

public class ConfigHandler {
    public ConfigHandler() {}
    
    public void createDefaultConfigs() {
        CreateDirectory(getConfigPath());
        createProjectsJsonIfNotExists();
    }

    private void createProjectsJsonIfNotExists() {
```
Whitespace and `camelCase` for methods